pages/vomnibar.coffee

```diff
@@ -222,7 +238,7 @@ class BackgroundCompleter
     id = BackgroundCompleter.messageId += 1
     @filterPort.onMessage.addListener handler = (msg) =>
```

Comment:
Yes.  I'm slowly trying to tidy up some of the legacy issues with this part of the code.  I think some of it may have been written quite some time ago.